requirements/edx/base.txt

```diff
@@ -1157,7 +1156,7 @@ wrapt==1.14.1
     # via
     #   -r requirements/edx/paver.txt
     #   deprecated
-xblock[django]==1.6.2
+xblock[django]==1.2.9
```

Comment:
This downgrade looks unrelated to the rest of the PR. Perhaps it was a rebase issue?

One way to check would be to start fresh with the latest master, run `make compile-requirements`, and see if this remains as `xblock[django]==1.6.2`.

Reply:
@kdmccormick It seems to be coming from one of the dependencies of xblock-poll. With latest master, it remains `xblock[django]==1.6.2` but when I add `xblock-poll` in base.in, it gets downgraded. I'll try to find the dependency causing this and correct it.

Reply:
@kdmccormick This should be fixed now.